Atomic/syscontainers.py

```diff
@@ -847,7 +846,6 @@ def extract_system_container(self, img, destination):
         repo = self._get_ostree_repo()
         if not repo:
             return False
-        self.args.remote = None
         return self._checkout_system_container(repo, img, img, 0, False, destination=destination, extract_only=True)
```

Comment:
the `_checkout_system_container` call here misses the argument `remote=self.args.remote`.

Everything else LGTM